x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers.ts

```diff
@@ -273,7 +273,7 @@ export function getWaterfall(
     : [];
   const traceRoot = getTraceRoot(childrenByParentId);
   const duration = getDuration(orderedItems);
-  const traceRootDuration = traceRoot && traceRoot.transaction.duration.us;
+  const traceRootDuration = traceRoot ? traceRoot.transaction.duration.us : 0;
```

Comment:
Is this correct? If there is no `traceRoot` is the `traceRootDuration` then `undefined` or `0`? 
@ogupte Does this change have any downstream effects or what is the reason for the change?

Reply:
I rewrote this line when I removed getTraceRoot, not noticing the original line, so the logic change was unintentional. I don't believe it has any negative side effects, but i will revert it just to be safe.